frontend/javascripts/oxalis/workers/byte_arrays_to_lz4_base64.worker.ts

```diff
@@ -1,5 +1,5 @@
 import Base64 from "base64-js";
-import * as lz4wasm from "lz4-wasm";
+import * as lz4wasm from "lz4-wasm-nodejs";
```

Comment:
I think, this needs to be undone? or does bucket compression work in the browser with that module?

Reply:
Good catch. Done